courses/serializers/v2/courses.py

```diff
@@ -63,6 +66,7 @@ def get_duration(self, instance):
 
         return None
 
+    @extend_schema_field(str)
     def get_time_commitment(self, instance):
```

Comment:
```suggestion 
    def get_time_commitment(self, instance) -> str | None:
```
looks like a bunch of other properties should have `| None` too. (or be Optional?)